Fix runtime error due missing import

Traceback (most recent call last):
  File "/nix/store/4qqhr7g9ri3n7v6ik9ahlpvyp18wcn4p-git-repo-2.15.3/bin/.repo-wrapped", line 1339, in <module>
    main(sys.argv[1:])
  File "/nix/store/4qqhr7g9ri3n7v6ik9ahlpvyp18wcn4p-git-repo-2.15.3/bin/.repo-wrapped", line 1304, in main
    _Init(args, gitc_init=(cmd == 'gitc-init'))
  File "/nix/store/4qqhr7g9ri3n7v6ik9ahlpvyp18wcn4p-git-repo-2.15.3/bin/.repo-wrapped", line 607, in _Init
    _Clone(url, dst, opt.clone_bundle, opt.quiet, opt.verbose)
  File "/nix/store/4qqhr7g9ri3n7v6ik9ahlpvyp18wcn4p-git-repo-2.15.3/bin/.repo-wrapped", line 912, in _Clone
    if clone_bundle and _DownloadBundle(url, cwd, quiet, verbose):
  File "/nix/store/4qqhr7g9ri3n7v6ik9ahlpvyp18wcn4p-git-repo-2.15.3/bin/.repo-wrapped", line 860, in _DownloadBundle
    r = urllib.request.urlopen(url, context=ssl.create_default_context())
NameError: name 'ssl' is not defined
builder for '/nix/store/4hvds8fv8xmzlm86yg9cf1lj6hrya7sg-amdvlk-src.drv' failed with exit code 1

diff --git a/repo b/repo
index 8b05def..f394b3e 100755
--- a/repo
+++ b/repo
@@ -236,8 +236,9 @@ import optparse
 import re
 import shutil
 import stat
+import ssl
 import urllib.error
 import urllib.request
 
 if sys.version_info[0] == 3:
   import urllib.request
